plugin/plugin.go

```diff
@@ -389,7 +407,11 @@ func (p *OrmPlugin) generateConvertFunctions(message *generator.Descriptor) {
 		if getFieldOptions(field).GetDrop() {
 			continue
 		}
-		p.generateFieldConversion(message, field, true)
+		var ofield *Field
+		if ormable != nil {
```

Comment:
I guess this check is redundant since there is always should be an `ormable` and `getOrmable` fails if it doesn't exist for some reason.